x-pack/plugins/event_log/server/es/names.ts

```diff
@@ -22,10 +22,13 @@ export interface EsNames {
 export function getEsNames(baseName: string): EsNames {
   const eventLogName = `${baseName}${EVENT_LOG_NAME_SUFFIX}`;
   const eventLogNameWithVersion = `${eventLogName}${EVENT_LOG_VERSION_SUFFIX}`;
+  const eventLogPolicyName = `${
+    baseName.startsWith('.') ? baseName.substring(1) : baseName
+  }${EVENT_LOG_NAME_SUFFIX}`;
```

Comment:
nit: `-policy` suffix can be part of the `eventLogPolicyName` variable and removed from below.